Mathlib/Algebra/Ring/Subring/Basic.lean

```diff
@@ -384,6 +384,14 @@ theorem center_eq_top (R) [CommRing R] : center R = ⊤ :=
 instance : CommRing (center R) :=
   { inferInstanceAs (CommSemiring (Subsemiring.center R)), (center R).toRing with }
 
+/-- The center of isomorphic nonunital semirings are isomorphic. -/
+@[simps!] def centerCongr (e : R ≃+* S) : center R ≃+* center S :=
+  NonUnitalSubsemiring.centerCongr e
+
+/-- The center of a nonunital semiring is isomorphic to the center of its opposite. -/
```

Comment:
This is is arguably backwards compared to the type of the equiv; either the docstring or statement should be flipped, I guess deciding based on some existing equivs in mathlib.

Reply:
Let me change the names to `centerToMulOpposite` ...